permissions/plugin-doxygen.yml

```
@@ -7,3 +7,5 @@ paths:
   - "org/jenkins-ci/plugins/doxygen"
 developers:
   - "gbois"
```

A developer here also
```suggestion
developers:
  - "gbois"
  - "notmyfault"
```